docs/content/development/extensions-core/druid-lookups.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: doc_page
+---
+# Cached Lookup Module
+
+**Please note that this is an experimental module and the development/testing still at early stage. Feel free to try it and give us your feedback.**
+ 
+## Description
+This module provides a per-lookup caching mechanism for JDBC data sources.
+The main goal of this cache is to speed up the access to a high latency lookup sources and to provide a caching isolation for every lookup source.
+Thus user can define various caching strategies or and implementation per lookup, even if the source is the same.
```

Comment:
Could you include something about whether it is intended that users could use this together with lookups-cached-global? Would be good to answer questions like:

- Do they conflict with each other or can they be loaded simultaneously?
- Does it make sense to run them side by side while migrating from one to the other?
- Does it make sense to run them side by side long term?